Going live (#2525)
umbraco/UmbracoDocs: Umbraco-StarterKit/Going-live/index.md

# Going live

In this section, you can find all the information needed in order to publish your website to the world wide web.

## Domain/Hostname

### Configure DNS records

The first step is to set up your Hostname, you can do this from the portal.
This guide will show you how to bind your hostname to your Umbraco Uno environment where you see your environment.

It is recommended to set-up an ALIAS record for your root hostname (e.g. mysite.s1.umbraco.io), rather than an A record for the umbraco.io service IP address - 23.100.15.180.
Check with your DNS host or hostname provider for details on how to configure this for your hostnames.

### Add Hostname from the portal

- Click Settings
- Select Hostnames
- Once inside Hostnames, click "Add new hostname"
- Add your hostname to the field and select add
- To keep track of the process, you can refresh the page when it is done sucessfully it will say: Protected by LATCH  

### Umbraco LATCH

> Every Umbraco Project is protected by LATCH witch means that a TLS Certificate will be applied automatically.

Reviewer: ```suggestion
Every Umbraco Uno project is protected by Umbraco LATCH witch means that a TLS Certificate will be applied automatically.
```

> If you are interested in learning more about [LATCH](https://umbraco.com/products/umbraco-cloud/umbraco-latch/) this is a good place to start.

Reviewer: ```suggestion
If you are interested in learning more about Umbraco LATCH check out the [product page on Umbraco.com](https://umbraco.com/products/umbraco-cloud/umbraco-latch/).
```

> ### SMTP

Reviewer: ```suggestion
## SMTP
```

> In Umbraco Uno projects we take care of SMTP(Simple Mail Transfer Protocol). So that you dont have to do anything.

Reviewer: Would be good to add some information about why SMTP is necessary - or, what it is used for.

## SEO

> Setting up should be part of your general workflow for each of your nodes. This is to make your page the most visible in search engines as possible, all of the fields below should be filled out.

Reviewer: ```suggestion
Setting up SEO should be part of your general workflow when you create content. This is to make your page is the most visible in search engines as possible. All of the fields below should be filled out.
```

- Company Name
- Site Name
- Company Logo
- Phone Number
- Email
- Latitude
- Longitude

> You can reach the general SEO Under your home node > Settings > General.

Reviewer: ```suggestion
You can reach the general SEO under your home node > Settings > General.
```

> There are more documentation about SEO at these pages [General settings](../Uno-pedia/Settings/General-Settings/index.md/#SEO) and [Specific settings ](../Uno-pedia/Settings/Specific-Settings/index.md/#SEO) 

Reviewer: ```suggestion
There are more documentation about SEO at these pages: [General settings](../Uno-pedia/Settings/General-Settings/index.md/#SEO) and [Specific settings ](../Uno-pedia/Settings/Specific-Settings/index.md/#SEO) 
```

> ## Newsletter

Reviewer: I think this needs a bit more info about the Newsletter widget, and perhaps also a link to it?

You will need to choose if you use Mailchimp or Campaign Monitor, and then after that fill out your API Key and Default Subscriber ID.

> This is necessary otherwise your newsletter won't work.

Reviewer: ```suggestion
This is necessary for the Newsletter widget to work.
```

## Tracking and Access

> Tracking and Access, can be found in the Settings > General section, If you want to use any extensions such as Google maps or Google Analytics, This is where you put the API Keys.

Reviewer: ```suggestion
The Tracking and Access group can be found in the General section node in the Content section.

If you want to use any extensions such as Google maps or Google Analytics, this is where you put the API Keys.
```

:::note

> It is important to note that the AIzaSyDX76tv4LsYpvuS-ue5cl6-C_pKRauv5XI API Key for Google Maps should be changed to your own API Key, as this standard one will stop working eventually.

Reviewer: ```suggestion
In order to your to get the most out of the Google Maps API, that the API Key is be changed to your own API Key.
```
